libs/blocks/global-navigation/global-navigation.js

```diff
@@ -381,6 +381,7 @@ class Gnav {
 
     const toggle = toFragment`<button
       class="feds-toggle"
+      daa-ll="hamburgermenu|close|gnav"
```

Comment:
It is very likely that the `|gnav` part is inferred from the closest `daa-lh` parent, which in this case is `header.global-navigation`, which has a `daa-lh` value of `gnav|homepage_milo|target-smb|hp` on the homepage, for example. We have a similar approach for dropdown triggers - https://github.com/adobecom/milo/blob/main/libs/blocks/global-navigation/global-navigation.js#L608

Reply:
We should be able to test this one through Omnibug

Reply:
I checked the interaction and we need two changes:
* as @overmyheadandbody noted, we need to remove the `|gnav` part, it will be automatically added by the analytics code
* we need to account for the upcoming action, meaning that when the hamburger menu is closed, the daa-ll value should be `hamburgermenu|open` and when the menu is already open it should be `hamburgermenu|close`.